DataFormats/OnlineMetaData/interface/OnlineMetaDataRaw.h

```diff
@@ -0,0 +1,67 @@
+#ifndef DATAFORMATS_ONLINEMETADATA_ONLINEMETADATARAW_H
+#define DATAFORMATS_ONLINEMETADATA_ONLINEMETADATARAW_H
+
+//---------------------------------------------------------------------------
+//!  \class OnlineMetaDataRaw
+//!  \brief Structure of raw data from soft FED 1022
+//!
+//!  \author Remi Mommsen - Fermilab
+//---------------------------------------------------------------------------
+
+#include <cstdint>
+#include "EventFilter/FEDInterface/interface/fed_header.h"
+#include "EventFilter/FEDInterface/interface/fed_trailer.h"
+
+namespace onlineMetaData {
+
+  struct Luminosity_v1
+  {
+    const uint64_t timestamp;
+    const uint16_t lumiSection;
+    const uint16_t lumiNibble;
+    const float instLumi;
```

Comment:
does the member alignment map to some hardware header alignment?
If not, better move uint16_t members to the end

Reply:
In this case the raw data comes from s/w. However, it does not change the alignment (nor the data blob size.) Therefore, I prefer to keep it in the logical order.